qurro/q2/plugin_setup.py

```diff
@@ -10,33 +10,26 @@
 import qiime2.plugin
 import qiime2.sdk
 from qurro import __version__
-from ._method import supervised_rank_plot, unsupervised_rank_plot
-from qurro._parameter_descriptions import EXTREME_FEATURE_COUNT, TABLE
-from qiime2.plugin import Metadata, Properties, Int
+from ._method import differential_plot, loading_plot
+from qurro._parameter_descriptions import EXTREME_FEATURE_COUNT, TABLE, DEBUG
+from qiime2.plugin import Metadata, Properties, Int, Bool
 from q2_types.feature_table import FeatureTable, Frequency
-from q2_types.feature_data import FeatureData
+from q2_types.feature_data import FeatureData, Differential
```

Comment:
Is this new Differential type a new QIIME core type?

Reply:
Yep! It'll be added in Q2 2019.7.

A side effect of that is that this will completely break old versions of Qurro and Songbird when used with the newest Q2 version. But it's ultimately a good thing, since it'll mean many tools can use the Differential type without relying on Songbird to define it like before.